engine/src/main/java/org/terasology/rendering/opengl/OpenGLUtils.java

```diff
@@ -167,6 +167,20 @@ public static void renderFullscreenQuad() {
         glPopMatrix();
     }
 
+    /**
+     * First sets a viewport and then renders a quad filling it.
+     *
+     * @param x              an integer representing the x coordinate (in pixels) of the origin of the viewport.
+     * @param y              an integer representing the y coordinate (in pixels) of the origin of the viewport.
+     * @param viewportWidth  an integer representing the width (in pixels) the viewport.
+     * @param viewportHeight an integer representing the height (in pixels) the viewport.
+     */
+    // TODO: perhaps remove this method and make sure the viewport is set explicitely.
+    public static void renderFullscreenQuad(int x, int y, int viewportWidth, int viewportHeight) {
```

Comment:
This is from PostProcessor. I will add a TODO on top of this method for the time being, since it might be hard to merge other branches with nodes. But definitely we need to find a suitable name for this method.